cmd/bclient/fileutil/filelist.go

```diff
@@ -0,0 +1,101 @@
+// This package manages a list of files, and their checksums
+
+package fileutil
+
+import (
+	"crypto/md5"
+	"encoding/base64"
+	"fmt"
+	"github.com/antonholmquist/jason"
+	"io"
+	"os"
+)
+
+// Our underlying data structure
+
+type FileList struct {
+	Root  string
+	Files map[string]map[int64][]byte
+}
+
+// Create an empty FileList
+
+func New(root string) *FileList {
+	fl := new(FileList)
+	fl.Root = root
+	fl.Files = make(map[string]map[int64][]byte)
```

Comment:
another nice way to write this function is
    return &FileList{
        Root: root,
        Files: make(map[string]map[int64][]byte),
    }

more fyi.